autogpt_platform/backend/backend/executor/database.py

```diff
@@ -179,6 +189,14 @@ def get_service_type(cls):
     # Block error monitoring
     get_block_error_stats = _(d.get_block_error_stats)
 
+    # Library
+    list_library_agents = _(d.list_library_agents)
+    add_store_agent_to_library = _(d.add_store_agent_to_library)
+
+    # Store
+    get_store_agents = _(d.get_store_agents)
+    get_store_agent_details = _(d.get_store_agent_details)
+
```

Comment:
We need to stop complicating stuff by adding unused client functions; let's remove them.
There is an unsed summary email api as well being added here, let's remove it too in this PR.

Reply:
How is this supposed to work? This reads to me like I should expose in all these different functions the various things I’m going to call from an executor? Is that not the case?

Reply:
Some comments in the file would be useful because right now it just reads as a while bunch of things being set equal to themselves. I add the same functions to sync and async because I figure that 
- it’s automatically converted
- Having a different exposed surface for each would be weird

let me know if this is a flawed understanding

Reply:
It's correct, 
There are two components: the server itself, class DatabaseManager, and its clients (sync & async)
We are exposing the API only on the server, and we create a separate client so we can choose what we want to call.  There is no point in introducing the function unless it's used on the client, and it's not weird. 

Ideally, the client can be different for each use case, so we know the methods we are actually calling. But for now, we can stick to the shared one. But it would be helpful not to introduce something that is not used, so we can still map what function calls are actually used for each client. So refactoring it later won't be such a pain.

Reply:
Okay so if I want to add a new function it needs to go on the client I'm using it plus the server so the client has access to it?

Reply:
Correct

Reply:
I'll do all of these in a follow up to clean